RecoMuon/MuonIdentification/src/CSCTimingExtractor.cc

```diff
@@ -120,19 +116,16 @@ CSCTimingExtractor::fillTiming(TimeMeasurementSequence &tmSequence, reco::TrackR
   GlobalVector momv(mom.x(), mom.y(), mom.z());
   FreeTrajectoryState muonFTS(posp, momv, (TrackCharge)muonTrack->charge(), theService->magneticField().product());
 
-  // get the CSC segments that were used to construct the muon
-  std::vector<const CSCSegment*> range = theMatcher->matchCSC(*muonTrack,iEvent);
-
   // create a collection on TimeMeasurements for the track        
   std::vector<TimeMeasurement> tms;
-  for (std::vector<const CSCSegment*>::iterator rechit = range.begin(); rechit!=range.end();++rechit) {
+  for (std::vector<const CSCSegment*>::const_iterator rechit = segments.begin(); rechit!=segments.end();++rechit) {
```

Comment:
move to range based loop
(also below at L133)

Reply:
What's the point? Isn't it just a shorter form of the loop without any performance benefits?

Reply:
https://stackoverflow.com/a/10821946